apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -561,6 +561,8 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     /**
      * Show Revoke confirmation.
+     * TODO - Currently revoke functionality is disabled until proper backend support is provided for disabling
+     * @link https://github.com/wso2-enterprise/asgardeo-product/issues/2668
```

Comment:
Lets create relevant issues in product-is and link here.

Reply:
Fixed in f1f310d